src/css/components/_progress.scss

```diff
@@ -27,6 +27,12 @@
 /* We need an increased hit area on hover */
 .video-js .vjs-progress-control:hover .vjs-progress-holder { font-size: 1.666666666666666666em }
 
+/* Also show the current time tooltip */
+.video-js .vjs-progress-control:hover .vjs-play-progress:after {
+  display: block;
+  font-size: 0.6em;
```

Comment:
Ah, I see. On hover, you want it slightly smaller following the mouse, but if you want it always on, you want it slightly bigger.
The only issue is that if you turn it on manually to always show, on hover, the font-size will change and it will flicker.